.env.example

```diff
@@ -1,2 +1,3 @@
 GOOGLE_API_KEY="Sample Key"
 API_HOST=http://127.0.0.1:5000
+rootURL='open-event-frontend'
```

Comment:
No user would need to set rootUrl for local testing. So, remove it from here. it is only required for gh-pages deployment. So, just set it on travis. (See next comment on how)